Tests/sdk_release/create_content_pr.py

```diff
@@ -0,0 +1,151 @@
+import requests
+import json
+import sys
+import argparse
+import urllib3
+import time
+from pathlib import Path
+from distutils.util import strtobool
+from create_release import get_changelog_text
+from Tests.scripts.utils.log_util import install_logging
+from Tests.scripts.utils import logging_wrapper as logging
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+SLACK_CHANGELOG_FILE = 'CHANGELOG_SLACK.txt'
+SLACK_MERGE_PRS_FILE = 'SLACK_MERGE_PRS_REQUEST.txt'
+CONTENT_PR_NUMBER_FILE = 'CONTENT_PR.txt'
+
+TIMEOUT = 60 * 60  # 1 hour
+
+PRS_LIST_TEMPLATE = 'https://api.github.com/repos/demisto/{repo}/pulls'
+UPDATE_SDK_VERSION_WORKFLOW = 'https://api.github.com/repos/demisto/content/actions' \
+                              '/workflows/update-demisto-sdk-version.yml/dispatches'
+
+SLACK_MERGE_PRS_MESSAGE = 'Please merge the demisto-sdk and content pull requests:\n{sdk_pr}\n{content_pr}'
```

Comment:
some of the messages are python constants and some are in the .gitlab files, it's best to keep them in python. it's more readable.